youtube_dl/extractor/cba.py

```diff
@@ -0,0 +1,104 @@
+        }
+    }
+    _FORMATS = {
+        'audio/ogg': {'id': '1', 'ext': 'ogg', 'preference': 100},
+        'audio/mpeg': {'id': '2', 'ext': 'mp3', 'preference': 50}
+    }
+    _API_KEY = None
+
+    def __init__(self, *args, **kwargs):
+        try:
+            self._API_KEY = os.environ["CBA_API_KEY"]
```

Comment:
Ok. Haven't found any other extractors that need API keys - apparently i didn't look well enough. Do you mind if it falls back to the environment varaible if `--password` ist empty? 
This makes it imho much easier to integrate into bigger applications.